operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -598,6 +600,16 @@ private ResourceRequirements buildKafkaExporterResources(ManagedKafka managedKaf
         // custom authorizer configuration
         addKafkaAuthorizerConfig(managedKafka, config);
 
+        config.put("strimzi.authorization.custom-authorizer.partition-counter.max.partitions",
+                Optional.ofNullable(managedKafka.getSpec()).map(ManagedKafkaSpec::getCapacity).map(ManagedKafkaCapacity::getMaxPartitions).orElse(-1));
+        config.put("strimzi.authorization.custom-authorizer.partition-counter.timeout-seconds", 10);
+        config.put("strimzi.authorization.custom-authorizer.partition-counter.private-prefix",
```

Comment:
can we not create single property for this and `managedkafka.kafka.acl.private-prefix`?

Reply:
@rareddy I'm not 100% sure what you mean. This is picking up on the `managedkafka.kafka.acl.private-prefix` from the properties file here, and setting it on the config in the Kafka CR's `spec.kafka.config`. I don't see it otherwise being set anywhere there.

Reply:
I mean I see the two properties defining the "prefix" can they not be combined into a single property?

Reply:
@rareddy I might still be misunderstanding, but I think there is only a single property defining the prefix (although it exists in two different contexts, with different names):

1. The fleetshard operator has a property called `managedkafka.kafka.acl.private-prefix` defined in the application.properties file. This isn't used directly in code (or passed on to other components as-is), it's referenced in other fleetshard properties in the application.properties, for substitution.

2. The group authorizer plugin will (as part of https://github.com/bf2fc6cc711aee1a0c2a/kafka-group-authorizer/pull/21) expect to know what the prefix is, but that isn't exposed to it through the broker configs anywhere (I suppose it could parse something like `strimzi.authorization.custom-authorizer.acl.010`, pull out and clean up the topic part, and use that, but that doesn't seem right). I instead chose to add a new property for it, and have named it with a similar prefix to the other properties where it's used: `strimzi.authorization.custom-authorizer.partition-counter.private-topic-prefix` as set in this PR.

So there's one fleetshard operator property, and one broker property (set to the value of the fleetshard operator property, but given a different name). Does that make sense, or am I still not getting what you're saying?

Reply:
ok, that is fine with me having different properties for these two contexts. Since they were both portraying the same issue I thought if a common property can be derived among both but does not seem that critical to have.